pkg/cloud/volume_limits.go

```diff
@@ -532,7 +533,7 @@ var nvmeInstanceStoreVolumes = map[string]int{
 
 // https://aws.amazon.com/ec2/instance-types
 // Despite the dl1.24xlarge having Gaudi Accelerators describe instance types considers them GPUs as such that instance type is in this table
-// g5.48xlarge is not added to this table as it is in the maxVolumeLimits.
+// g5.48xlarge and p4d.24xlarge are not added to this table as it is in the maxVolumeLimits.
```

Comment:
We'll want to also update [hack/generate-gpu-count-table.sh](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/dba6d7090f33e5ccd1452214691e0d1433d7881a/hack/generate-gpu-count-table.sh#L38) and [generate-instance-store-table.sh](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/dba6d7090f33e5ccd1452214691e0d1433d7881a/hack/generate-instance-store-table.sh#L40) otherwise running those scripts will re-add the values.